alien-module/prototypes/recipe/alien-module.lua

```diff
@@ -233,7 +233,7 @@ data:extend({
 		result = "alien-fuel",
 		result_count = 1,
 		ingredients = {
-			{ "alien-plate", 2 },
+			{ "artifact-ore", 2 },
```

Comment:
please revert this change

Reply:
In my opinion, it doesn't make sense to melt down alien artifacts into alien plates and then burn them. It makes more sense to process the artefact directly into fuel.